src/WebEid.Security/Nonce/NonceGeneratorBuilder.cs

```diff
@@ -4,7 +4,7 @@ namespace WebEID.Security.Nonce
     using System.Security.Cryptography;
     using Cache;
 
-    public sealed class NonceGeneratorBuilder
+    public class NonceGeneratorBuilder
```

Comment:
Making `NonceGeneratorBuilder` `sealed` made sense as it is not meant to be extended.